packages/cli/package.json

```diff
@@ -118,7 +118,7 @@
     "assets": [
       "package.json",
       "**/openapi-template.yaml",
-      "appmap.html"
+      "built/appmap.html"
```

Comment:
I thought it was cleaner to retain the same directory structure that we publish to npm. The fact that we had to add this file to the root directory seemed more like a bug.